back/api/locale/fr/LC_MESSAGES/django.po

```diff
@@ -72,485 +72,523 @@ msgstr "Recherche"
 msgid "street"
 msgstr "adresse"
 
-#: api/models.py:33
+#: api/models.py:34
 msgid "street2"
 msgstr "complément d'adresse"
 
-#: api/models.py:34
+#: api/models.py:35
 msgid "postcode"
 msgstr "NPA"
 
-#: api/models.py:35
+#: api/models.py:36
 msgid "city"
 msgstr "ville"
 
-#: api/models.py:36
+#: api/models.py:37
 msgid "country"
 msgstr "pays"
 
-#: api/models.py:38
+#: api/models.py:39
 msgid "company_name"
 msgstr "raison sociale"
 
-#: api/models.py:39
+#: api/models.py:40
 msgid "phone"
 msgstr "téléphone"
 
-#: api/models.py:56
+#: api/models.py:57
 msgid "belongs_to"
 msgstr "appartient à"
 
-#: api/models.py:57 api/models.py:126
+#: api/models.py:58 api/models.py:127
 msgid "sap_id"
 msgstr "Numéro SAP"
 
-#: api/models.py:58 api/models.py:135
+#: api/models.py:59 api/models.py:136
 msgid "subscribed"
 msgstr "utilisateur permanent"
 
-#: api/models.py:59
+#: api/models.py:60
 msgid "is_active"
 msgstr "actif"
 
-#: api/models.py:63
+#: api/models.py:64
 msgid "contact"
 msgstr "contact"
 
-#: api/models.py:71 api/models.py:156
+#: api/models.py:72 api/models.py:168
 msgid "copyright"
 msgstr "copyright"
 
-#: api/models.py:81 api/models.py:98 api/models.py:109 api/models.py:148
-#: api/models.py:237 api/models.py:282
+#: api/models.py:82 api/models.py:99 api/models.py:110 api/models.py:155
+#: api/models.py:250 api/models.py:295
 msgid "name"
 msgstr "nom"
 
-#: api/models.py:83 api/models.py:679
+#: api/models.py:84 api/models.py:718
 msgid "link"
 msgstr "lien"
 
-#: api/models.py:84
+#: api/models.py:85
 msgid "Please complete the above URL"
 msgstr "Veuillez compléter l'URL ci-dessus"
 
-#: api/models.py:91
+#: api/models.py:92
 msgid "document"
 msgstr "document"
 
-#: api/models.py:102 api/models.py:581 api/models.py:714
+#: api/models.py:103 api/models.py:596 api/models.py:753
 msgid "data_format"
 msgstr "format du produit"
 
-#: api/models.py:113
+#: api/models.py:114
 msgid "order type"
 msgstr "type de commande"
 
-#: api/models.py:114
+#: api/models.py:115
 msgid "order types"
 msgstr "types de commande"
 
-#: api/models.py:125
+#: api/models.py:126
 msgid "user"
 msgstr "utilisateur"
 
-#: api/models.py:127 api/models.py:729
+#: api/models.py:128 api/models.py:768
 msgid "ide_number"
 msgstr "Numéro IDE"
 
-#: api/models.py:130 api/models.py:732
+#: api/models.py:131 api/models.py:771
 msgid "IDE number is not valid"
 msgstr "Le numéro IDE n'est pas valable"
 
-#: api/models.py:133
+#: api/models.py:134
 msgid "contract_accepted"
 msgstr "Date d'acceptation du contrat"
 
-#: api/models.py:134
+#: api/models.py:135
 msgid "is_public"
 msgstr "public?"
 
-#: api/models.py:136
+#: api/models.py:137
 msgid "birthday"
 msgstr "Date de naissance"
 
-#: api/models.py:140
+#: api/models.py:141
 msgid "identity"
 msgstr "identité"
 
-#: api/models.py:147
+#: api/models.py:150
+#, fuzzy
+#| msgid "is_public"
+msgid "Public"
+msgstr "Public"
+
+#: api/models.py:151
+msgid "Approval needed"
+msgstr "Diffusion restreinte"
```

Comment:
Is that the right translation?

Reply:
Yep, how would you say it for a data that needs to be validated by the data owner before it's processed?